js/data.js

```diff
@@ -11,3 +11,10 @@ window.addEventListener('beforeunload', function () {
   var JSONdata = JSON.stringify(data);
   localStorage.setItem('code-journal', JSONdata);
 });
+
+var previousEntriesJSON = localStorage.getItem('code-journal');
+
+if (previousEntriesJSON !== null) {
+  data.entries = JSON.parse(previousEntriesJSON).entries;
+  data.view = JSON.parse(previousEntriesJSON).view;
```

Comment:
You should just reassign the value of `data` to the parsed value, you want every property of the `data` object to be preserved between usages.